api/src/main/java/org/motechproject/nms/api/web/OpsController.java

```diff
@@ -175,5 +178,20 @@ public GetBookmarkResponse getBookmarkWithScore(@RequestParam(required = false)
         log("RESPONSE: /ops/getbookmark", String.format("bookmark=%s", ret.toString()));
         return ret;
     }
+
+    @RequestMapping(value = "/releaseNumber",
+            method = RequestMethod.POST,
+            headers = { "Content-type=application/json" })
+    @ResponseStatus(HttpStatus.OK)
+    @Transactional
+    public void releaseNumber(@RequestBody DeactivateSubscriptionContract deactivateSubscriptionContract) {
```

Comment:
Also, this contract is a little 'heavy' for an ops task that just needs a phone number for now. You want to keep in mind that your support team should just be able to curl a post request to this api and pass a phone number to it, as a query parameter.

Otherwise, they will have to build a json request and that just adds more complexity to this feature that isn't required.